content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -118,12 +118,15 @@ PRISMA_MIGRATION_ENGINE_BINARY=custom/my-migration-engine-unix
 
 ### <inlinecode>PRISMA_QUERY_ENGINE_BINARY</inlinecode>
 
-`PRISMA_QUERY_ENGINE_BINARY` is used to set a custom location for your own query engine binary.
+`PRISMA_QUERY_ENGINE_BINARY` is used to set a custom location for your own query engine binary. 
 
 ```terminal
 PRISMA_QUERY_ENGINE_BINARY=custom/my-query-engine-unix
 ```
 
+For Prisma CLI it allows you to define the query engine file to be used.  
+For Prisma Client on build time (during `prisma generate`) it defines where the query engine file will be copied from into the Prisma Client, on run time (when using the generated Client) it can be used to define the specific  query engine file to be used instead of the included one.
```

Comment:
```suggestion
For Prisma Client, on build time (during `prisma generate`), it defines where the query engine file will be copied from into the Prisma Client. At run time (when using the generated Client) it can be used to define the specific query engine file to be used instead of the included one.
```